css.html.markdown

```diff
@@ -252,6 +252,48 @@ recommended that you avoid its usage.
 * `B` is next.
 * `D` is the last one.
 
+## Media Queries
+
+CSS Media Queries are a feature in CSS 3 which allows you to specify when certain CSS rules should be applied, such as when printed, or when on a screen with certain dimensions or pixel density.
```

Comment:
Could we add a note that media queries do not add to the specificity? That was a bit surprising, to me at least.